src/shared/components/checkbox/styles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+import Checkbox from "@mui/material/Checkbox";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormGroup from "@mui/material/FormGroup";
+
+export const Wrapper = styled(Checkbox)``;
+
+export const Container = styled(FormGroup)``;
```

Comment:
O padrão que está sendo utilizado no projeto é de Wrapper ser o nome do component que engloba todos os outros elementos. Inverter o Container com Wrapper. Além disso, o faz mais sentido nomear o Container de Checkbox ou CheckboxControl por exemplo

```suggestion
export const Wrapper = styled(FormGroup)``;

export const CheckboxControl = styled(Checkbox)``;
```